src/Workspaces/Remote/Core/ExternalAccess/UnitTesting/Api/UnitTestingAssetStorageWrapper.cs

```diff
@@ -7,13 +7,13 @@
 
 namespace Microsoft.CodeAnalysis.ExternalAccess.UnitTesting.Api
 {
-    internal readonly struct UnitTestingAssetStorageWrappper
```

Comment:
❔ Are you not currently using this type? If this type is already in use, renaming it may require a coordinated insertion.

Reply:
It is in used. There is another PR in testimpact which rename the usage of this